bin/partner-provision.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env sh
```

Comment:
I'm not sure whether this belongs in `bin` or `tools`. Ping @dereksmart for input.